src/core/prometheus.cc

```diff
@@ -634,6 +635,7 @@ future<> write_text_representation(output_stream<char>& out, const config& ctx,
 class metrics_handler : public handler_base  {
     sstring _prefix;
     config _ctx;
+    static std::function<bool(const mi::labels_type&)> _false_function;
```

Comment:
Maybe use seastar::noncopyable_function for a tiny better performance?

Reply:
sure

Reply:
It actually needs to be copied, becasue it returned as a result from a function